qa/tasks/cephfs/mount.py

```diff
@@ -120,6 +142,11 @@ def create_files(self):
                 'sudo', 'touch', os.path.join(self.mountpoint, suffix)
             ])
 
+    def test_create_file(self, filename='testfile', dirname=None, user=None,
```

Comment:
We'll definitely want to test file creates, but we need to test the entire range of operations that require permissions checks. These are primarily done via the Client::inode_permission methods in libcephfs. The various filesystem operations call the may_* functions that call inode_permission with various MAY_* flags.

We'll want to ensure that all of the MAY_* flag combinations to inode_permission are tested. It doesn't need to be done in this first patchset, but we do need to ensure that the entire range of dir ops are tested -- creates, deletes, renames, link, unlink. We'll also need to ensure that opening files is tested in various O_* flag and ACL combinations.

That said, I'm fine with merging an initial set like this and then extending it in follow-on sets.